codegen/java-gen/src/test/resources/tests/replicated-entity/multimap-unnamed-new-style/generated-managed/com/example/replicated/multimap/MultiMapServiceEntityProvider.java

```diff
@@ -70,4 +71,9 @@ public final MultiMapServiceEntityRouter newRouter(ReplicatedEntityContext conte
       SomeMultiMapDomain.getDescriptor()
     };
   }
+  
+  @Override
+  public Serializer serializer() { 
+    return Serializer.noopSerializer();
+  }
```

Comment:
Seems noisy that the default empty/no serializer must be in each generated class, can't there be a default impl be in the base Provider class/trait instead? (And only generate override when needed)

Reply:
Agree. I started with a default impl before I had the generators, but then moved for not clear reason (that I remember). I will bring it back.